src/BUILD

```diff
@@ -537,7 +537,8 @@ sh_binary(
         "tmp/build",
         "--platform",
     ] + select({
-        "//src/conditions:darwin": ["darwin"],
```

Comment:
Hrm yea still not sure if you need Darwin here, I think you do even if it points to the x86 version still

Reply:
Could you elaborate on why we might want to contine selecting more generally on "darwin"?

AFAIK this script is only invoked in a buildkite pipeline that (after this change) will be run on both darwin_x86_64 and darwin_arm64.

Reply:
so after https://github.com/bazelbuild/bazel/pull/17547 this won't be possible, but today my understanding is that on a x86_64 mac, if you don't pass any CPU and you're using the CLT toolchain, you will get `darwin` as a CPU. If you explicitly pass `--cpu=darwin_x86_64` then you won't ever hit it, but that weirdness is the goal of my other PR. but we have to merge this, then re-merge my toolchain PR, and then the darwin CPU PR before that is true

Reply:
I see, I don't think that should be an issue in this case. We won't be using any flags or the CLT toolchain.